packages/fuels/package.json

```diff
@@ -44,11 +45,22 @@
     "@fuel-ts/sparsemerkle": "workspace:*",
     "@fuel-ts/testcases": "workspace:*",
     "@fuel-ts/transactions": "workspace:*",
-    "@fuel-ts/wallet": "workspace:*",
+    "@fuel-ts/versions": "workspace:*",
     "@fuel-ts/wallet-manager": "workspace:*",
-    "@fuel-ts/wordlists": "workspace:*"
+    "@fuel-ts/wallet": "workspace:*",
+    "@fuel-ts/wordlists": "workspace:*",
+    "commander": "^9.4.1"
```

Comment:
It should not affect browsers in any way because `commander` is used only inside the `cli.ts` file, which is never imported by any other module except the app binary entry path (`bin.ts`); both of which are never referenced in any other place inside the package.